include/irr/asset/IAssetManager.h

```diff
@@ -148,7 +148,7 @@ class IAssetManager : public core::IReferenceCounted
 
         const IGeometryCreator* getGeometryCreator() const;
         const IMeshManipulator* getMeshManipulator() const;
-        const IGLSLCompiler* getGLSLCompiler() const { return m_glslCompiler.get(); }
+        IGLSLCompiler* getGLSLCompiler() const { return m_glslCompiler.get(); }
```

Comment:
why is it not working with const anymore?

Reply:
because there must be a way to `getGLSLCompiler()->getIncludeHandler()->addBuiltinIncludeLoader(...)`

Reply:
I see.